src/me/mrCookieSlime/Slimefun/Android/ProgrammableAndroid.java

```diff
@@ -713,14 +714,18 @@ private void mine(Block b, Block block) {
 		Collection<ItemStack> drops = block.getDrops();
 		if (!blockblacklist.contains(block.getType()) && !drops.isEmpty() && CSCoreLib.getLib().getProtectionManager().canBuild(UUID.fromString(BlockStorage.getLocationInfo(b.getLocation(), "owner")), block)) {
 			SlimefunItem item = BlockStorage.check(block);
+			boolean allow = false;
 			if (item != null) {
 				if (fits(b, item.getItem())) {
 					if (SlimefunItem.blockhandler.containsKey(item.getID())) {
-						if (SlimefunItem.blockhandler.get(item.getID()).onBreak(null, block, item, UnregisterReason.ANDROID_DIG)) {
-							pushItems(b, BlockStorage.retrieve(block));
-							if (SlimefunItem.blockhandler.containsKey(item.getID())) SlimefunItem.blockhandler.get(item.getID()).onBreak(null, block, item, UnregisterReason.ANDROID_DIG);
-							block.getWorld().playEffect(block.getLocation(), Effect.STEP_SOUND, block.getType());
-							block.setType(Material.AIR);
+						if (allow) {
```

Comment:
"allow" is never changed, this will always be false.

Reply:
Though it remained false, it still prevented the items from being destroyed. The only items it would destroy are vanilla items. I understand that you wish for me to try another way of implementing this fix, but that is what worked out for me.

Reply:
And there we go, that should do it.